### DISCUSSION TITLE
I need help to setup SAMD21

### DISCUSSION THREAD

**Discussion by Bexin3** [tttapa/Control-Surface#874] [Q&A]
Hi, Id want to use something like the example teensy code, to get audio data sent to Arduino zero so it can then forward it using I2S

**Comment by tttapa** [comment]
I'm afraid this is beyond the scope of this library, the main focus is MIDI, not audio. Audio is handled by the [Teensy Audio library](https://www.pjrc.com/teensy/td_libs_Audio.html).
